docs/going-into-production.rst

```diff
@@ -102,143 +154,149 @@ their type. For instance:
+A good DNS label for this node might be ``node-01-md``.
+
+Here, ``node`` is used as base label with a sequence number of ``01``. Every
+node in the cluster should have a unique sequence number, independent of the
+node type. The letters ``md`` indicate that this node has ``node.master`` and
+``node.data`` set to ``true``.
+
+If you optionally want your node name to match (:ref:`see above
+<node-name-match>`), configure the `node.name`_ setting in your
+`configuration`_ file, like so:
 
 .. code-block:: yaml
 
     node.name: node-01-md
 
-Here, ``node`` is a base name with a sequence number of ``01``. Every node in
-the cluster should have a unique sequence number, independent of the node type.
-The letters ``md`` indicate that this node has ``node.master`` and
-``node.data`` set to ``true``.
+Alternatively, you can configure this setting at startup with a command-line
+option:
+
+.. code-block:: console
+
+    sh$ bin/crate \
+            -Cnode.name=node-01-md
 
 
 .. _prod-node-d:
 
 Request handling and query execution nodes
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-You can configure a node that only handles client requests and query execution
-(i.e., is not master-eligible) like this:
+You can `configure`_ a node that only handles client requests and query
+execution (i.e., is not master-eligible) like this:
 
 .. code-block:: yaml
 
     node.master: false
     node.data: true
 
-A good name for this node might be:
+A good DNS label for this node might be ``node-02-d``.
+
+Here, ``node`` is used as base label with a sequence number of ``02``. Every
+node in the cluster should have a unique sequence number, independent of the
+node type. The letter ``d`` indicates that this node has ``node.data`` set to
+``true``.
+
+If you optionally want your node name to match (:ref:`see above
+<node-name-match>`), configure the `node.name`_ setting in your
+`configuration`_ file, like so:
 
 .. code-block:: yaml
 
     node.name: node-02-d
 
-Here, ``node`` is a base name with a sequence number of ``02``. Every node in
-the cluster should have a unique sequence number, independent of the node type.
-The letter ``d`` indicates that this node has ``node.data`` set to ``true``.
+Alternatively, you can configure this setting at startup with a command-line
+option:
+
+.. code-block:: console
+
+    sh$ bin/crate \
+            -Cnode.name=node-02-d
 
 
 .. _prod-node-m:
 
 Cluster management nodes
-~~~~~~~~~~~~~~~~~~~~~~~~
+^^^^^^^^^^^^^^^^^^^^^^^^
 
-You can configure a node that handles cluster management (i.e., is
+You can `configure`_ a node that handles cluster management (i.e., is
 master-eligible) but does not handle query execution loads like this:
 
 .. code-block:: yaml
 
     node.master: true
     node.data: false
 
-A good name for this node might be:
+A good DNS label for this node might be ``node-03-m``.
+
+Here, ``node`` is used as base label with a sequence number of ``02``. Every
```

Comment:
``02`` should be ``03``

Reply:
pushed a fixup. thx @infoverload

Reply:
The node name isn't necessary a DNS label is it?

e.g.:
node name: node-03-m
hostname: node03.cluster.crate.io